src/common/appConfig.ts

```diff
@@ -0,0 +1,11 @@
+import { AppConfig } from '../components/publicApi/types'
+
+let exportConfig: AppConfig = {
+  termsOfServiceSite: 'https://edge.app/terms-of-service/'
+}
+export const setAppConfig = (appConfig: AppConfig | undefined): void => {
+  if (appConfig == null) return
+  exportConfig = { ...appConfig }
```

Comment:
Why do you not just assign appConfig to the `exportConfig`?

You could instead use `Object.assign` if you want to export `exportConfig` as `appConfig` directly with no getter (`getAppConfig`).

Reply:
Because if the caller of `setAppConfig` mutated their passed in appConfig object it would mutate the exported config. Mutation of the appConfig should REQUIRE a call to `setAppConfig`. Object.assign would be equivalent to spread operator so no reason why one vs the other except for preference.